helper/funcs.go

```diff
@@ -33,6 +33,12 @@ var invalidFilenameNonASCII = regexp.MustCompile(`[[:^ascii:]/\\<>:"|?*]`)
 // invalidFilenameStrict = invalidFilename plus additional punctuation
 var invalidFilenameStrict = regexp.MustCompile(`[/\\<>:"|?*$()+=[\];#@~,&']`)
 
+type Copyable[T any] interface {
+	any
```

Comment:
Never seen this; what does requiring `any` do here?

Reply:
Whoops! Nothing! This was my first generic interface rodeo and it took me a few tries. Fixed. :sweat_smile: